step-06/js/main.js

```diff
@@ -160,8 +160,10 @@ function signalingMessageCallback(message) {
 
   } else if (message.type === 'candidate') {
     peerConn.addIceCandidate(new RTCIceCandidate({
+      sdpMLineIndex: message.label,
+      sdpMid: message.id,
       candidate: message.candidate
-    }));
+    }));// Firefox compatible
```

Comment:
No need to add "Firefox compatible". We want to make it compatible with all browsers.